src/Arcus.Templates.AzureFunctions.Databricks.JobMetrics/.template.config/ide.host.json

```diff
@@ -1,8 +1,15 @@
 {
   "$schema": "http://json.schemastore.org/vs-2017.3.host",
-  "learnMoreLink": "https://templates.arcus-azure.net/features/web-api-template",
+  "name": {
+    "text": "Arcus Azure Functions Databricks Job Metrics project template"
+  },
+  "description": {
+    "text": "Provide a template to easily build an Azure Functions that reports the completed Databricks job runs as metrics."
+  },
+  "learnMoreLink": "https://templates.arcus-azure.net/features/azurefunctions-databricks-jobmetrics-template",
   "icon": "icon.jpg",
+  "isApi": true,
```

Comment:
This is not an API
```suggestion
  "isApi": false,
```

Reply:
Ah, right, it's a timer. I'll remove the line then all together.